pkg/apis/hwameistor/v1alpha1/localdiskclaim_types.go

```diff
@@ -72,6 +72,9 @@ type DiskClaimDescription struct {
 
 	// Capacity of the disk in bytes
 	Capacity int64 `json:"capacity,omitempty"`
+
+	// LocalDiskName for specifying LocalDisk
+	LocalDiskName string `json:"localDiskName,omitempty"`
```

Comment:
should be array of LocalDiskName?

Reply:
It's a good idea, but it may be conflict with `NodeName` property. So in my view, `localDiskName` is fine.

Reply:
it may be possible to specify multiple LocalDisk per node, right?